handlers/main.yml

```diff
@@ -1,11 +1,17 @@
 ---
 # handlers file for wildfly
 
+- name: systemd reload
```

Comment:
Is there any particular reason you want to add yet another handler instead of using `block:` in [here](https://github.com/inkatze/wildfly/blob/master/tasks/configure.yml#L41)?

Reply:
I can change handler for task in configuration.yml. This task will reinit systemd. I think that the result will be the same.

Reply:
yes it *will* be the same but I think using block is more comprehensible.